Snakefile

```diff
@@ -125,85 +129,73 @@ def clock_rate_std_dev(w):
     return clock_rate_std_dev[w.subtype][w.time]
 
 
-rule download:
-    message: "Downloading sequences from fauna"
+rule download_sequences:
+    output:
+        sequences = "data/{segment}/sequences.fasta.zst",
+    params:
+        s3_src=S3_SRC,
+    shell:
+        """
+        aws s3 cp {params.s3_src:q}/{wildcards.segment}/sequences.fasta.zst {output.sequences}
+        """
+
+rule download_metadata:
     output:
-        sequences = "data/{subtype}_{segment}.fasta"
+        metadata = "data/metadata.tsv.zst",
     params:
-        fasta_fields = "strain virus accession collection_date region country division location host domestic_status subtype originating_lab submitting_lab authors PMID gisaid_clade h5_clade",
-        download_by = download_by
+        s3_src=S3_SRC,
     shell:
         """
-        python3 {path_to_fauna}/vdb/download.py \
-            --database vdb \
-            --virus avian_flu \
-            --fasta_fields {params.fasta_fields} \
-            --select  {params.download_by} locus:{wildcards.segment} \
-            --path data \
-            --fstem {wildcards.subtype}_{wildcards.segment}
-        """
-### comment
-rule parse:
-    message: "Parsing fasta into sequences and metadata"
+        aws s3 cp {params.s3_src:q}/metadata.tsv.zst {output.metadata}
+        """
+
+rule filter_sequences_by_subtype:
     input:
-        sequences = rules.download.output.sequences
+        sequences="data/{segment}/sequences.fasta.zst",
+        metadata="data/metadata.tsv.zst",

```

Comment:
In Zika and measles workflows both sequences and metadata are decompressed before moving forward with further Augur functions. Here, for usability, my personal preferences would be to at least decompress metadata. I'm looking at this all the time while checking on a build and in this current setup I'll have to manually run `zstd -d metadata.tsv.zst` each time to look at this. Currently, the uncompressed version totals 8.9Mb, so I don't think it would be an issue in terms of disk space. 

I'm okay with keeping sequences compressed by default and only manually decompressing if it becomes necessary.

Reply:
This choice was my local bias in usage; [VisiData](https://www.visidata.org/) automatically decompressed zstandard files (when [the zstandard Python library](https://pypi.org/project/zstandard/) is installed), so opening the compressed metadata works with `vd data/metadata.tsv.zst`. That plus the fact that `augur filter` also handles these files made me not think twice about keeping them compressed.

I've switched back to uncompressed for the more general usability and consistency with our other workflows in 0d56cb8.